scripts/ots-app.js

```diff
@@ -20,6 +20,13 @@ jQuery.noConflict();
           }
         });
     },
+
+    dropdownOpen : function() {
+      $('.dropdown').on('click', 'a', function(event){
+        var target = event.target
```

Comment:
Are we going to use [JavaScript Standard Style](https://standardjs.com/)?  Looks like some Semicolons are missing or omitted on purpose. Guess its better to stick with inserting them after a statement to ensure consistency 😉